### ISSUE TITLE
Image segmentation does not work

### ISSUE THREAD

**Issue by alvaroiba** [conscienceli/VoTT-label-export#1]
Hi,

I'm using Vott to label regions on image files. I try to convert vott-json to mask image files, but your code does not work. It seems to be prepared to work only on video files.

By the other hand, Vott creates a project json with .vott extension and every simple image has their own xxxx-asset.json file. It is not covered in the current project. Isn't it?

Thanks in advance

**Comment by conscienceli** [comment]
Yes. Currently, it is only designed for videos because we only use videos in our project.

Maybe I can add the support for image segmentation next month.
